### PR TITLE
1990e realign

### PR DESCRIPTION
**Pull request by cvalarida**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1695

Additional Benefits
> image

Removed from School Selection
> image

Sponsor Information
> image

> image

How does that look, @goldenmeanie?

### CODE REVIEW

File: src/js/edu-benefits/pages/additionalBenefits.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    title: 'Additional Benefits',
+    initialData: {},
+    uiSchema: {
+      'ui:order': fields,
+      nonVaAssistance: {
+        'ui:title': 'Are you getting, or do you expect to get any money (including, but not limited to, federal tuition assistance) from the Armed Forces or public health services for any part of your coursework or training?',
+        'ui:widget': 'yesNo'
+      },
+      civilianBenefitsAssistance: {
+        'ui:title': 'I am receiving benefits from the U.S. Government as a civilian employee during the same time as I am seeking benefits from VA.'
```
Comment (jbalboni) [inline src/js/edu-benefits/pages/additionalBenefits.js]: A PR that I just did for 5490 changed this field to a Yes/No and bolded "civilian employee." I think the latter should be done here, but not not sure about the former.

File: src/js/edu-benefits/1990e/config/form.js [hunk 2]
```diff
@@ -94,33 +96,33 @@ const formConfig = {
...
       pages: {
         sponsorVeteran: {
-          title: 'Sponsor Veteran',
+          title: 'Sponsor Information',
           path: 'sponsor-veteran',
           uiSchema: {
             veteranFullName: {
               first: {
-                'ui:title': 'Veteran first name'
+                'ui:title': 'Sponsor first name'
```
Comment (jbalboni) [inline src/js/edu-benefits/1990e/config/form.js]: Could we import the common fullName uiSchema definition and override the labels (with _.merge)? That way we don't have to specify that class name again (and makes it easier if we have common changes later).

**Review by jbalboni — APPROVED** [review]
Couple comments, but looks good.

File: src/js/edu-benefits/pages/additionalBenefits.js (hunk 1 above)
Comment (cvalarida) [inline src/js/edu-benefits/pages/additionalBenefits.js]: Yeah, when we merge the two together, we should take what you've got and put it in here, methinks.

File: src/js/edu-benefits/1990e/config/form.js (hunk 2 above)
Comment (cvalarida) [inline src/js/edu-benefits/1990e/config/form.js]: Sure. While I'm here, I can do that. :+1:

Comment (cvalarida) [inline src/js/edu-benefits/1990e/config/form.js]: Sure. While I'm here I can do that. :+1:

**Pull request merged**